setup.cfg

```diff
@@ -31,6 +31,7 @@ install_requires =
     scipy
     zarr
     numba
+    typing-extensions
```

Comment:
This doesn't need to go into requirements.txt or the mypy requirements?

Reply:
@eric-czech it's already in the `requirements.txt` (thus this bug), but good point about `mypy requirements`, missed that one. We should really automated this via https://github.com/pystatgen/sgkit/issues/39

Reply:
Fixed